lib/config.js

```diff
@@ -1,14 +1,21 @@
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-import bedrock from 'bedrock';
-const {config} = bedrock;
+import {config} from '@bedrock/core';
 
 const namespace = 'bedrock-did-resolver-http';
 const cfg = config[namespace] = {};
 
-// support did:key and veres one keys by default
-cfg.supportedMethods = ['key', 'v1'];
+// a list of which did:methods we supported
+// experimental is for tests only
+cfg.supportedMethods = ['key', 'v1', 'experimental'];
```

Comment:
```suggestion
cfg.supportedMethods = ['key', 'v1', 'experimental'];
```

We definitely don't want the `experimental` DID Method in any of our resolution code -- this is going to go into production and could become an attack vector. Why do we need this here?

Reply:
We a method that is not key for testing the `did:key` statement about the method always being `key`. We could use v1, but this actually shows a larger issue, right now passing in a `did:v1` will result in this implementation throwing as it only allows `did:key`.